### PR TITLE
[1LP][RFR]  Adding new test of service retire manual approval

### PR DESCRIPTION
**Pull request by niyazRedhat**
{{ pytest: cfme/tests/services/test_service_manual_approval.py -k 'test_service_retire_manual_approval' --long-running --use-template-cache -sqvvv }}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 04d185123a68b184afae6c3d1122eac399c1cfd6

The *local* fixture **`service_retirement_request`** is used in the following files:
- **cfme/tests/services/test_service_manual_approval.py**
    - *test_service_retire_manual_approval*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit ddc83b5f32e03b406aa21c0a9c2f2af849231421

The *local* fixture **`service_retirement_request`** is used in the following files:
- **cfme/tests/services/test_service_manual_approval.py**
    - *test_service_retire_manual_approval*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
Lint report for commit d5db2ebf5e3a832cef014f154fca5f94977449f8:

`cfme/tests/services/test_service_manual_approval.py`:
- :red_circle: Line 11:1: **F401** *'cfme.services.myservice.ui.MyServiceDetailView' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by niyazRedhat** [comment]
Its getting skipped in 5.11 beacuse of this bz https://bugzilla.redhat.com/show_bug.cgi?id=1769934

File: cfme/tests/services/test_service_manual_approval.py [hunk 1]
```diff
@@ -87,3 +113,55 @@ def test_service_manual_approval(appliance, provider, modify_instance,
...
+        testSteps:
+            1. Add Service Catalog, Order the Service
+            2. Retire the service
+            3. Navigate to Service retirement request Details page
+            4. Manually approve the service retirement request
+        expectedResults:
+            1.
+            2.
+            3.
+            4. Admin should able to approve the retirement request
```
Comment (sshveta) [inline cfme/tests/services/test_service_manual_approval.py]: ** Admin should be able to approve the retirement request

File: cfme/services/myservice/ui.py [hunk 2]
```diff
@@ -250,23 +250,24 @@ def is_displayed(self):
...
-            message='Service Retirement wait')
-    else:
-        request_descr = f"Service Retire for: {self.name}"
-        service_request = self.appliance.collections.requests.instantiate(
-            description=request_descr)
-        service_request.wait_for_request()
+    if wait:
+        if self.appliance.version < '5.10':
+            wait_for(
+                lambda: view.entities.lifecycle.get_text_of('Retirement State') == 'Retired',
```
Comment (ganeshhubale) [inline cfme/services/myservice/ui.py]: We are not testing for 5.9. So we can remove this condition: ```self.appliance.version < '5.10':```

File: cfme/tests/services/test_service_manual_approval.py [hunk 3]
```diff
@@ -87,3 +113,55 @@ def test_service_manual_approval(appliance, provider, modify_instance,
...
+    """
+    service, _ = service_vm
+    with appliance.context.use(ViaUI):
+        service.retire(wait=False)
+        request_description = 'Service Retire for: {name}'.format(name=service.name)
+        service_request = appliance.collections.requests.instantiate(
```
Comment (ganeshhubale) [inline cfme/tests/services/test_service_manual_approval.py]: I would use fstring here.

File: cfme/tests/services/test_service_manual_approval.py [hunk 4]
```diff
@@ -87,3 +113,55 @@ def test_service_manual_approval(appliance, provider, modify_instance,
...
+        view = appliance.browser.create_view(RequestDetailsToolBar)
+        if not view.approve.is_displayed and BZ(1721479, forced_streams=['5.10', '5.11']).blocks:
+            navigate_to(appliance.server, "Dashboard")
+            service_request.approve_request(method='ui', reason="Approved")
+        else:
+            service_request.approve_request(method='ui', reason="Approved")
+        assert service_request.row.approval_state.text == 'Approved'
+        service_request.wait_for_request()
+        msg = "Request failed with the message {}".format(service_request.rest.message)
+        assert service_request.is_succeeded(), msg
```
Comment (ganeshhubale) [inline cfme/tests/services/test_service_manual_approval.py]: @niyazRedhat 
Can we add finalizer for ```service_request``` removal?

**Comment by dajoRH** [comment]
I detected some fixture changes in commit c93d7de662e01de99faafb2ad1cf1356a7bc6921

The *local* fixture **`service_retirement_request`** is used in the following files:
- **cfme/tests/services/test_service_manual_approval.py**
    - *test_service_retire_manual_approval*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Pull request merged**